dashboard/test/ui/features/cookie_dcdo.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Cookie DCDO
+  Scenario: Cookie DCDO flag is set
+    Given I am on "http://studio.code.org/users/sign_in"
+    Then element "script[data-dcdo]" attr "data-dcdo" includes ""cookie_dcdo_test":null"
+
+    # Tests assigning of a cookie DCDO flag
+    When I set the DCDO key "cookie_dcdo_test" to "assigning_works"
```

Comment:
I'd like this step to be more descriptive of what it's actually doing. What about something like,

```suggestion
    When I use a cookie to mock the DCDO key "cookie_dcdo_test" as "assigning_works"
```

Or even just,

```suggestion
    When I mock the DCDO key "cookie_dcdo_test" as "assigning_works"
```